src/main/java/sg/ncl/MainController.java

```diff
@@ -2003,16 +2017,50 @@ public String joinTeamAppAwaitingApproval(Model model) {
     }
     
     //--------------------------Get List of scenarios filenames--------------------------
-    private List<String> getScenarioFileNameList() {
-		List<String> scenarioFileNameList = new ArrayList<String>();
-		File[] files = new File(SCENARIOS_DIR_PATH).listFiles();
-		for (File file : files) {
-			if (file.isFile()) {
-				scenarioFileNameList.add(file.getName());
-			}
-		}
+    private List<String> getScenarioFileNameList() throws WebServiceRuntimeException {
+        logger.info("Retrieving scenario file names");
+//        List<String> scenarioFileNameList = null;
+//        try {
+//            scenarioFileNameList = IOUtils.readLines(getClass().getClassLoader().getResourceAsStream("scenarios"), StandardCharsets.UTF_8);
+//        } catch (IOException e) {
+//            throw new WebServiceRuntimeException(e.getMessage());
+//        }
+//        File folder = null;
+//        try {
+//            folder = new ClassPathResource("scenarios").getFile();
+//        } catch (IOException e) {
+//            throw new WebServiceRuntimeException(e.getMessage());
+//        }
+//        List<String> scenarioFileNameList = new ArrayList<>();
+//		File[] files = folder.listFiles();
+//		for (File file : files) {
+//			if (file.isFile()) {
+//				scenarioFileNameList.add(file.getName());
+//			}
+//		}
+        // FIXME: hardcode list of filenames for now
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)